v22.1/backup-architecture.md

```diff
@@ -0,0 +1,96 @@
+
+The ultimate aim of the job creation phase is to complete all of these checks and write the detail of what the backup job should complete to a _job record_.
+
+If a [`detached`](backup.html#detached) backup was requested, the `BACKUP` statement is complete as it has created an uncommitted, but otherwise ready-to-run backup job. You'll see the job ID returned as output. Without the `detached` option, the job is committed and the statement waits for the backup job to start, run, and terminate to then return the results.
+ 
+Once the job record is committed, the cluster will try to run the backup job even if a client disconnects or the node handling the `BACKUP` statement terminates. From this point, the backup is a persisted job that any node in the cluster can take over executing to ensure it runs. The job record will move to the system jobs table ready for adoption. 
+
+## Resolution phase 
+
+Once one of the nodes has claimed the job from the system job table, it will take the job record’s information and outline a plan. This node becomes the _coordinator_. In our example, **Node 2** becomes the coordinator and starts to complete the following to prepare and resolve the targets for distributed backup work: 
+
+- Test the connection to the storage bucket URL (`'s3://bucket'`).
+- Map out the directory to which the nodes will write.
+- Calculate the keys and time range of the backup data.
+- Determine the [leaseholder](architecture/overview.html#architecture-leaseholder) nodes for the keys to backup. 
+- Provide a plan to the leaseholder nodes.
+
+To map out the storage location's directory to which the nodes will write the data, the coordinator identifies the [type](backup-and-restore-overview.html#backup-and-restore-types) of backup. This determines the name of the new (or edited) directory to store the backup files in. For example, if there is a full backup existing in the target storage location, the upcoming backup will be [incremental](take-full-and-incremental-backups.html#incremental-backups) and therefore append to the full backup. For more information on how CockroachDB structures backups in storage, see [Backup collections](take-full-and-incremental-backups.html#backup-collections).
+
+### Key and time range resolution
+
+In this part of the resolution phase, the coordinator will calculate all the necessary spans of keys and their time range that the cluster needs to export for this backup. It divides the key spans based on which node is the range [leaseholder](architecture/overview.html#architecture-leaseholder) for that key. For this backup, each leaseholder node is responsible for:
```

Comment:
based on which node is *the leaseholder of* the range for that key *span* ?

also it isn't technically the leaseholder who is responsible, for the following steps but rather a backup sql processor which is running on the node which is the leaseholder, or maybe on a different node e.g. in serverless or if things moved after planning.